interop-tests/src/arch.rs

```diff
@@ -114,7 +116,11 @@ pub(crate) mod native {
                 )
                 .map(|(peer_id, conn), _| (peer_id, StreamMuxerBox::new(conn)))
                 .boxed(),
-                format!("/ip4/{ip}/udp/0/webrtc-direct"),
+                // IPv4 does not pass tests locally, but IPv6 does.
```

Comment:
No error logs, it just wasn't connecting. I will keep investigating, it might be a WebRTC issue or just me having made a mistake somewhere. I've noted in the meta description at the top and will follow up

Reply:
Chrome. I want to try this again using ip4 to see if I run into the same issues still.

Reply:
I tried this again using IPv4 with WebRTC and it hangs up again. The test just stops usually with no error. Although once I got this in the log:

```sh
[1693331139.378][SEVERE]: bind() returned an error: Only one usage of each socket address (protocol/network address/port) is normally permitted. (0x2740)
```

I immediately changed WebRTC back to IPv6 and it passed the interop tests fine locally. CI might be another result?

It could be a localhost issue or a odd WebRTC issue, I am not sure. Maybe the OS is trying to assign the same port due to `Protocol::Udp(0)`? Or maybe it's just because `IPv4` is legacy and we really shouldn't be using it anymore and this is the universe trying to tell us just that 🤷‍♂️🌌

Reply:
I just tried this locally and it worked for me, resolving here.